src/uniprotkb/config/UniProtKBColumnConfiguration.tsx

```diff
@@ -703,33 +703,67 @@ UniProtKBColumnConfiguration.set(UniProtKBColumn.ccInteraction, {
     const interactionComments = data[EntrySection.Interaction].commentsData.get(
       'INTERACTION'
     ) as InteractionComment[];
-    return (
-      <ExpandableList displayNumberOfHiddenItems>
-        {interactionComments?.map((interactionCC) =>
-          interactionCC.interactions.map((interaction) => (
-            <div
-              key={
-                interaction.type === InteractionType.SELF
-                  ? 'self'
-                  : `${interaction.interactantOne.uniProtKBAccession}-${interaction.interactantTwo.uniProtKBAccession}`
-              }
-            >
-              {interaction.type === InteractionType.SELF
-                ? 'Itself'
-                : interaction.interactantOne.uniProtKBAccession && (
-                    <Link
-                      to={getEntryPath(
-                        Namespace.uniprotkb,
-                        interaction.interactantOne.uniProtKBAccession
-                      )}
-                    >
-                      {interaction.interactantOne.uniProtKBAccession}
-                    </Link>
+    const { primaryAccession } = data;
+
+    const interactionElements = interactionComments
+      ?.map((interactionCC) =>
+        interactionCC.interactions.map((interaction) => {
+          if (interaction.type === InteractionType.SELF) {
+            return 'self';
+          }
+          if (
+            interaction.interactantOne.uniProtKBAccession ===
+              primaryAccession &&
+            interaction.interactantTwo.uniProtKBAccession
+          ) {
+            return (
+              <div
+                key={`${interaction.interactantOne.uniProtKBAccession}-${interaction.interactantTwo.uniProtKBAccession}`}
+              >
+                <Link
+                  to={getEntryPath(
+                    Namespace.uniprotkb,
+                    interaction.interactantTwo.uniProtKBAccession
                   )}
-            </div>
-          ))
-        )}
-      </ExpandableList>
+                >
+                  {interaction.interactantTwo.uniProtKBAccession}
```

Comment:
I see the entry type is in the data, maybe it would be good to display the reviewed/unreviewed icon maybe through the `src/shared/components/results/AccessionView.tsx` component?

Reply:
Any opinion on this then?

Reply:
Forgot to comment on this - I haven't seen "type" as a reviewed/unreviewed indication (at least for P05067), but it should be about the interaction type (binary, xeno or self). However I noticed that this information is not properly propagated from the flatfile, so contacted Leo and Jie about it and they are going to look into it as it should be there.

Reply:
> I see the entry type is in the data
Ignore that, this is when I looked in the devtools, the alignment of the field names made me think it was part of the interaction comment I was seeing, but it's actually on the top-level entry.